images.txt

```diff
@@ -1,7 +1,7 @@
 docker.elastic.co/eck/eck-operator:2.8.0
 docker.elastic.co/elasticsearch/elasticsearch:8.10.1
 docker.io/bitnami/postgresql:11.14.0-debian-10-r28
-docker.io/busybox:1.32
+docker.io/library/busybox:1.32
```

Comment:
the `library` being added automatically right?

Reply:
I run the update-images - so yes